src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -7312,6 +7337,7 @@ export class StudyViewPageStore
         }
     >(
         q => ({
+            await: () => [this.filtersRemote],
```

Comment:
We wait for `filtersRemote` but we don't use it in `invoke`, is that because you want to trigger the `filtersRemote` to fetch data?

Reply:
Yes, I added this because we need `this.filters` to be completely loaded, which in turn relies on the async `this.selectedDriverTiersMap`